### ISSUE TITLE
V12.0 Suggestion > New help page template

### ISSUE THREAD

**Issue by gregory-moka** [SythilTech/Odoo#85]
Hello,
I worked on the help page template, and I would like to give you the code if you want to your future version.
I used your initial suggestion to make it match with Odoo default section and template.
Let me know if you are interrested, and how could I give you the sources.
<hr>

**Support page :** 
![image](https://user-images.githubusercontent.com/45974412/58790289-5ff44500-85f0-11e9-8068-4a400e56649f.png)

**Groupe page :** 
![image](https://user-images.githubusercontent.com/45974412/58790398-9af67880-85f0-11e9-96a9-952d64a17481.png)

**Help page :** 
![image](https://user-images.githubusercontent.com/45974412/58790472-bbbece00-85f0-11e9-988f-1b46e179b03c.png)

**Comment by SythilTech** [comment]
Wow that is definitely very fine, it would be good to see how it handles help groups without images, overall it is very pleasing to the eyes, you are free to make a pull request if you want to be credited for your work.

**Comment by gregory-moka** [comment]
**Without image :** 
You're right, we should define 2 different limit to the lists, one with picture, one without, in order not to have to much lost spaces :
![image](https://user-images.githubusercontent.com/45974412/58857379-67295a80-86a5-11e9-997b-b0c4e69693b2.png)

**Comment by SythilTech** [comment]
A placeholder image could be used to create a consistent spacing.
An customise option to turn images on and off may also be helpfully for companies that don't want to bother sourcing / creating for each help group.

**Comment by gregory-moka** [comment]
Or, user should think about using image or not. If he is able to create image, he does for all groups, if not, no pictures.

**Comment by SythilTech** [comment]
Yes, generally it will be an all or nothing situation, overall the current design is fine since  I don't expect people to mix and match non image and image help groups.

**Comment by ghost** [comment]
great work :-)

**Comment by gregory-moka** [comment]
Hi, 
I have finished my update of your app, and i want to make a pull request on your project. Can you explain how and where do I have to do it ? I'm blocked on the git pull command.
Thanks :)

**Comment by gokhanerdogdu** [comment]
The design seems very good. Will you merge these changes?

**Comment by gregory-moka** [comment]
Hope it will be done very soon :)
https://github.com/SythilTech/Odoo/pull/90
